t/cobrand/bexley.t

```diff
@@ -261,6 +261,16 @@ FixMyStreet::override_config {
         $mech->get_ok('/report/' . $report->id);
         $mech->content_contains('Report ref:&nbsp;' . $report->id);
     };
+
+    subtest 'phishing warning is shown for new reports' => sub {
+        $mech->log_out_ok;
+        $mech->get_ok('/report/new?longitude=0.15356&latitude=51.45556&category=Lamp+post');
```

Comment:
Does this test need a `$mech->content_contains('if asked for personal information, please do not respond');` like the one below?